lang/cs/Org.Apache.REEF.IO.TestClient/HadoopFileInputPartitionTest.cs

```diff
@@ -161,5 +163,26 @@ public IEnumerable<byte> Deserialize(string fileFolder)
                 }
             }
         }
+
+        /// <summary>
+        /// Read bytes from all the files in the set and return one by one
+        /// </summary>
+        /// <param name="filePaths"></param>
+        /// <param name="local"></param>
+        /// <returns></returns>
+        public IEnumerable<byte> Deserialize(ISet<string> filePaths, bool local)
+        {
+            foreach (var f in filePaths)
+            {
+                using (FileStream stream = File.Open(f, FileMode.Open))
+                {
+                    BinaryReader reader = new BinaryReader(stream);
+                    while (reader.PeekChar() != -1)
+                    {
+                        yield return reader.ReadByte();
```

Comment:
It is a test. In the test, I only put a few bytes in total. 
A real case would either an entire file or a big chuck.